### ISSUE TITLE
Incorrect ordering of `R` and `S` in `TransactionSnapshot.Of`

### ISSUE THREAD

**Issue by DavePearce** [Consensys/besu-sequencer-plugins#945]
Currently, we have this in `TransactionSnapshot.java` (see [here](https://github.com/Consensys/linea-tracer/blob/2b1b5bc51496862c96cd81d28919c467994f2bc8/arithmetization/src/main/java/net/consensys/linea/blockcapture/snapshots/TransactionSnapshot.java#L58)):

```
public record TransactionSnapshot(
    String r,
    String s,
    String v,
    ...
)

public static TransactionSnapshot of(Transaction tx) {
    return new TransactionSnapshot(
        tx.getS().toString(16),
        tx.getR().toString(16),
```

The ordering of `S` and `R` appears to be wrong in the `of` function.

**Comment by DavePearce** [comment]
The fix is simple: swap the calls `getS()` and `getR()`.  Since this is on the capture side, it then needs to be deployed before any effect can be observed.  All captured blocks thus far have swapped values for `r` and `s` in the json.  Could either manually swap them or just recapture them, etc.

**Comment by DavePearce** [comment]
Fixed by #946
